CONTRIBUTING.md

```diff
@@ -14,6 +14,9 @@ are important to the project's success.
     * [Contact us](#discussion) before starting significant work.
     * IMPORTANT: For new libraries, [get permission from the library owner first](#adding-a-new-library).
     * Create your stubs [conforming to the coding style](#stub-file-coding-style).
+    * Add tests for your stubs to `test_data`. These tests should are
```

Comment:
Thanks! I've created a separate section in CONTRIBUTING.md taking your comments into account.